tools/src/launcher/Main.hs

```diff
@@ -325,7 +326,9 @@ clientScenario logConf node wallet updater nodeTimeout report = do
   where
     killNode nodeHandle nodeAsync = do
         logInfo "Killing the node"
-        liftIO $ Process.terminateProcess nodeHandle
+        liftIO (tryAny (Process.terminateProcess nodeHandle)) >>= \case
```

Comment:
Why do we need to catch any exception? Isn't it only normal `IOException` we need to catch?